stetho-sample/src/debug/java/com/facebook/stetho/sample/APODDumperPlugin.java

```diff
@@ -0,0 +1,115 @@
+package com.facebook.stetho.sample;
+
+import android.content.ContentResolver;
+import android.database.Cursor;
+import com.facebook.stetho.dumpapp.ArgsHelper;
+import com.facebook.stetho.dumpapp.DumpException;
+import com.facebook.stetho.dumpapp.DumpUsageException;
+import com.facebook.stetho.dumpapp.DumperContext;
+import com.facebook.stetho.dumpapp.DumperPlugin;
+
+import java.io.PrintStream;
+import java.util.Iterator;
+import java.util.List;
+
+public class APODDumperPlugin implements DumperPlugin {
+
+  private static final String NAME = "apod";
+
+  private static final String CMD_LIST = "list";
+  private static final String CMD_CLEAR = "clear";
+  private static final String CMD_REMOVE_ROW = "delete";
+  private static final String CMD_REFRESH = "refresh";
+
+  private final ContentResolver mContentResolver;
+  private final APODRssFetcher mAPODRssFetcher;
+
+  public APODDumperPlugin(ContentResolver contentResolver) {
+    mContentResolver = contentResolver;
+    mAPODRssFetcher = new APODRssFetcher(mContentResolver);
+  }
+
+  @Override
+  public String getName() {
+    return NAME;
+  }
+
+  @Override
+  public void dump(DumperContext dumpContext) throws DumpException {
+    PrintStream writer = dumpContext.getStdout();
+    Iterator<String> argsIter = dumpContext.getArgsAsList().iterator();
+
+    String command = ArgsHelper.nextOptionalArg(argsIter, null);
+
+    if (CMD_LIST.equalsIgnoreCase(command)) {
+      doList(writer);
+    } else if (CMD_REMOVE_ROW.equalsIgnoreCase(command)) {
+      doRemove(writer, argsIter);
+    } else if (CMD_CLEAR.equalsIgnoreCase(command)) {
+      doClear(writer);
+    } else if (CMD_REFRESH.equalsIgnoreCase(command)) {
+      doRefresh(writer);
+    } else {
+      usage(writer);
+      if (command != null) {
+        throw new DumpUsageException("Unknown command: " + command);
+      }
+    }
+  }
+
+  private void doList(PrintStream writer) {
+    Cursor cursor = mContentResolver.query(
+        APODContract.CONTENT_URI,
+        null /* projection */,
+        null /* selection */,
+        null /* selectionArgs */,
+        APODContract.Columns._ID);
+
+    int count = 0;
+
+    while (cursor.moveToNext()) {
+      writer.println(String.format("Row #%d", count++));
+      for (int i = 0; i < cursor.getColumnCount(); ++i) {
+        writer.println(String.format("  %s: %s", cursor.getColumnName(i), cursor.getString(i)));
+      }
+    }
+
+    writer.println();
+  }
+
+  private void doRemove(PrintStream writer, Iterator<String> argsIter) throws DumpUsageException {
+    String rowId = ArgsHelper.nextArg(argsIter, "Expected rowId");
```

Comment:
`Expected <row-id>`.  Be consistent with your help, not with your internal code.